db/init_db.js

```diff
@@ -38,10 +51,10 @@ async function createTables() {
       "firstName" VARCHAR(255) NOT NULL,
       "lastName" VARCHAR(255) NOT NULL,
       email TEXT UNIQUE NOT NULL,
-      imageURL VARCHAR(255),
+      imageURL VARCHAR(255) DEFAULT 'http://www.freeimageslive.com/galleries/sports/moods_emotions/preview/happy_face.jpg',
```

Comment:
`"imageURL"` instead of `imageURL`, or just `imageurl`

Reply:
You might want to say `NOT NULL`, so that if nothing is inserted, if we attempt to insert `null`, sql (I think) should not accept null, and instead insert this default.